### ISSUE TITLE
What's validation?

### ISSUE THREAD

**Issue by thevooght** [CSAILVision/semantic-segmentation-pytorch#234]
Dear all,
I know this is a very simple question: but isn't validation used to tune hyperparameters rather than giving accuracy and IoU?
I learned to evaluate on a test set to determine accuracy, but ADE20K did not seem to have a test set with the predicted responses. Is there somebody who can explain me this a bit better?

Thanks a lot,
Cedric

**Comment by ShihuaiXu** [comment]
同问, 代码不对验证集做在线验证的吗, 是否得自己写这段代码
